app/scripts/modules/core/src/presentation/formsy/components/Input.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+import { FormComponent, IFormComponentState, IFormComponentProps } from '../FormComponent';
+
+export interface IProps extends IFormComponentProps {
```

Comment:
This is exported because Typescript requires it, but it is not re-exported from the module index:

```
export { Input } from './Input';
export { TextArea } from './TextArea';
```

cool?